### PR TITLE
Add background to Launchpage and simplify background code 

### PR DESCRIPTION
**Pull request by Megumiiiiii**

### CODE REVIEW

**Comment by abdelhamidbakhta** [comment]
@fricoben @0xChqrles can you review please ?

File: frontend/src/pages/Home/index.tsx [hunk 1]
```diff
@@ -10,10 +10,7 @@ import * as styles from './style.css'
 export default function HomePage() {
   return (
     <Box>
-      <Box as="span" className={styles.backgroundContainer}>
-        <Box as="span" className={styles.background} />
-      </Box>
-
+      <Box as="span" className={`${styles.backgroundContainer} ${styles.background}`} />
```
Comment (0xChqrles) [inline frontend/src/pages/Home/index.tsx:13]: We should use
```js
className={clsx(styles.backgroundContainer, styles.background)}
```

File: frontend/src/pages/Launch/index.tsx [hunk 2]
```diff
@@ -63,6 +63,7 @@ export default function LaunchPage() {
 
   return (
     <Row className={styles.wrapper}>
+      <Box as="span" className={`${styles.backgroundContainer} ${styles.background}`} />
```
Comment (0xChqrles) [inline frontend/src/pages/Launch/index.tsx:66]: Not sure we want this background in the app too, imo it's better to keep a clean an simple app interface

File: frontend/src/components/NavBar/index.tsx [hunk 3]
```diff
@@ -13,7 +13,7 @@ export const links = [
     path: '/launch',
   },
   {
-    name: 'Manager',
+    name: 'Manage',
```
Comment (0xChqrles) [inline frontend/src/components/NavBar/index.tsx:16]: I prefer to keep an "action incentiver" wording with `Deploy` `Manage` `Launch` etc.

File: frontend/src/pages/Launch/style.css.ts
Comment (0xChqrles) [inline frontend/src/pages/Launch/style.css.ts:1]: Some goes here, plus there's a lot of code duplication from HomePage style

File: frontend/src/pages/Home/index.tsx (hunk 1 above)
Comment (Megumiiiiii) [reply frontend/src/pages/Home/index.tsx]: edited

**Review by abdelhamidbakhta — APPROVED** [review]
lgtm

**Comment by abdelhamidbakhta** [comment]
are you on the telegram group @Megumiiiiii ?

**Comment by Megumiiiiii** [comment]
Nope, where is it. i can't find it on onlydust

**Comment by Megumiiiiii** [comment]
thanks🫡

**Pull request merged**